src/test/java/befaster/solutions/HLO/HelloSolutionTest.java

```diff
@@ -1,4 +1,4 @@
-package befaster.solutions.SUM;
+package befaster.solutions.HLO;
 
```

Comment:
Would have liked to have seen some unit tests, possibly TDDing some of these new classes into existence. The ShoppingCart in particular seemed like a good candidate for that.